src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/integration/wiremock/ExternalUsersApiMockServer.kt

```diff
@@ -1956,6 +1956,28 @@ class ExternalUsersApiMockServer : WireMockServer(WIREMOCK_PORT) {
     )
   }
 
+  fun stubNoUsersFoundForRolesUsername(username: String) {
+    stubFor(
+      get("/users/username/$username/roles")
+        .willReturn(
+          aResponse()
+            .withStatus(NOT_FOUND.value())
+            .withHeaders(HttpHeaders(HttpHeader("Content-Type", "application/json")))
+            .withBody(
+              """
+                {
+                  "status": 404,
+                  "errorCode": null,
+                  "userMessage": "User not found: Account for username $username not found",
+                  "developerMessage": "Account for username $username not found",
+                  "moreInfo": null
+                } 
+              """.trimIndent()
+            )
+        )
+    )
+  }
```

Comment:
we already have stubNoUsersFoundForUsername - maybe we should combine these and pass in the url?

Reply:
have added this query to a separate ticket HAAR-1223

Reply:
Updated this stub